### PR TITLE
Added homework_3

### PR DESCRIPTION
**Pull request by YuriyChaban**
Added homework_3

### CODE REVIEW

File: homeworks/yuriy.chaban_YuriyChaban/homework_3/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  "name": "homework_3",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "YuriyChaban",
+  "license": "ISC"
+}
```
[the remark this answers is not in the document]
Comment (markoffden) [reply homeworks/yuriy.chaban_YuriyChaban/homework_3/package.json]: please remove package.json from commit?

[the remark this answers is not in the document]
Comment (YuriyChaban) [reply homeworks/yuriy.chaban_YuriyChaban/homework_3/package.json]: Sorry, will be done in 5 minutes

File: homeworks/yuriy.chaban_YuriyChaban/homework_3/task2.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    if (Number.isNaN(min) || Number.isNaN(max)) {
+        console.log('\nYour min range or max range is not a valid range.');
+    }
+}
+
+function getNumberOfDegreeTwo(min, max) {
+    const arrayOfNumber = [];
+    for (let i = min; i < max; i++) {
```
Comment (markoffden) [inline homeworks/yuriy.chaban_YuriyChaban/homework_3/task2.js]: why not `i <= max`? `max` also can be one of the needed numbers

**Comment by YuriyChaban** [comment]
I've change the code, you can check when you will have time

**Pull request merged**